### PR TITLE
Add eprint function

### LINKED ISSUE #610: Allow users to emit errors.
**Is your feature request related to a problem? Please describe.**
Create a `eprint` function.
Right now when a user proactively checks for errors (which is our recommended best practice) the errors can only be tracked in `text_output`. Now that we have errors and the UI is rendering error'd tasks we should allow proactive checks to print to the error channel.

**Describe the solution you'd like**
Implement a `eprint` function that writes text to the errors channel.

**Describe alternatives you've considered**
Na

**Additional context**
Na

### PR DESCRIPTION
**Pull request by hulto**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
- [x] Adds `eprint` function to allow users to emit errors manually.
- [x] Move `eprint` to a native function
- [ ] Support `eprint` in golem interactive.

#### Which issue(s) this PR fixes:

Fixes #610

### CODE REVIEW

File: implants/lib/eldritch/src/runtime/eprint_impl.rs [hunk 1]
```diff
@@ -0,0 +1,65 @@
+use crate::runtime::{messages::ReportErrorMessage, Environment};
+use anyhow::Result;
+
+pub fn eprint(env: &Environment, message: String) -> Result<()> {
+    env.send(ReportErrorMessage {
+        id: env.id(),
+        error: message.clone(),
```
Comment (KCarretto) [inline implants/lib/eldritch/src/runtime/eprint_impl.rs:7]: unnecessary clone, move the `eprintln!` above this and borrow `&message` in that call

File: implants/lib/eldritch/src/runtime/eval.rs [hunk 2]
```diff
@@ -142,6 +142,16 @@ pub struct Runtime {
     rx: Receiver<Message>,
 }
 
+#[starlark_module]
+fn error_handler(builder: &mut GlobalsBuilder) {
+    #[allow(unused_variables)]
+    fn eprint(starlark_eval: &mut Evaluator<'v, '_>, message: String) -> anyhow::Result<NoneType> {
+        let env = crate::runtime::Environment::from_extra(starlark_eval.extra)?;
+        eprint_impl::eprint(env, message)?;
```
Comment (KCarretto) [inline implants/lib/eldritch/src/runtime/eval.rs:150]: Love that we pass env here, greatly demonstrates the separation between eprint (starlark) and eprint_impl (native rust)

**Review by KCarretto — APPROVED** [review]

File: implants/lib/eldritch/src/runtime/eprint_impl.rs (hunk 1 above)
Comment (hulto) [reply implants/lib/eldritch/src/runtime/eprint_impl.rs:7]: I cannot.
error expects a String not &String
and we use message later in `eprintln!`

**Comment by hulto** [comment]
Ignoring golem interactive for now tracking here:
https://github.com/spellshift/realm/issues/625

**Pull request merged**